Cargo.toml

```diff
@@ -8,4 +8,8 @@ name = "kvproto"
 
 [dependencies]
 protobuf = "1.2.1"
+futures = "0.1"
```

Comment:
Yes, https://github.com/stepancheg/grpc-rust#use-compiled-protos-in-your-project